src/OpenTelemetry/Metrics/Builder/MeterProviderBuilderExtensions.cs

```diff
@@ -303,14 +303,29 @@ public static MeterProviderBuilder ConfigureResource(this MeterProviderBuilder m
             return meterProviderBuilder;
         }
 
+        /// <summary>
+        /// Run the given actions to initialize the <see cref="MeterProvider"/>.
+        /// </summary>
+        /// <param name="meterProviderBuilder"><see cref="MeterProviderBuilder"/>.</param>
+        /// <returns><see cref="MeterProvider"/>.</returns>
+        public static MeterProvider? Build(this MeterProviderBuilder meterProviderBuilder)
+        {
+            if (meterProviderBuilder is MeterProviderBuilderBase meterProviderBuilderBase)
+            {
+                return meterProviderBuilderBase.InvokeBuild();
+            }
+
+            return null;
+        }
+
         /// <summary>
         /// Sets the <see cref="ExemplarFilter"/> to be used for this provider.
         /// This is applied to all the metrics from this provider.
         /// </summary>
         /// <param name="meterProviderBuilder"><see cref="MeterProviderBuilder"/>.</param>
         /// <param name="exemplarFilter"><see cref="ExemplarFilter"/> ExemplarFilter to use.</param>
         /// <returns>The supplied <see cref="MeterProviderBuilder"/> for chaining.</returns>
-        public static MeterProviderBuilder SetExemplarFilter(this MeterProviderBuilder meterProviderBuilder, ExemplarFilter exemplarFilter)
+        internal static MeterProviderBuilder SetExemplarFilter(this MeterProviderBuilder meterProviderBuilder, ExemplarFilter exemplarFilter)
```

Comment:
Really minor nit which you could just say no to 😆, but instead of moving the `Build` method you could just `#pragma warning disable` this guy for the time being. The Build method is the last method in the corresponding file for tracer/logger providers, so might forget to move it back.

Reply:
Done!